.github/workflows/deploy-tests.yml

```
@@ -47,7 +47,7 @@ jobs:
   run-dev-tests:
     needs: assess-file-changes
     if: ${{ needs.assess-file-changes.outputs.SOURCE_CHANGED == 'true' }}
-    uses: catalystneuro/neuroconv/.github/workflows/dev-testing.yml@main
+    uses: catalystneuro/neuroconv/.github/workflows/dev-testing.yml@try_to_add_dandi_to_dev
```

I checked and the test is there. let's revert this to main before we merge.
```suggestion
    uses: catalystneuro/neuroconv/.github/workflows/dev-testing.yml@main
```